### PR TITLE
github: use yml template

### PR DESCRIPTION
**Pull request by hi-rustin**
### What problem does this PR solve? 

None
### What is changed and how it works?

use yml template to get better experience
### Check List 

Tests 
 - No code

Code changes

None

Side effects

None

Related changes

None
### Release note 

```release-note

None
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/question.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: "\U0001F914 General Question"
+description: Usage question that isn't answered in docs or discussion
+labels: ["question"]
```
Comment (amyangfei) [inline .github/ISSUE_TEMPLATE/question.yml:3]: no `textarea` for questions

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      label: What did you do?
+      description: If possible, provide a recipe for reproducing the error.
+    validations:
+      required: true
+  - type: textarea
+    id: expect
+    attributes:
+      label: What did you expect to see?
+    validations:
+      required: true
```
Comment (amyangfei) [inline .github/ISSUE_TEMPLATE/bug-report.yml:21]:
```suggestion
      required: false
```

File: .github/ISSUE_TEMPLATE/question.yml (hunk 1 above)
Comment (hi-rustin) [reply .github/ISSUE_TEMPLATE/question.yml:3]: Added.

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 2 above)
Comment (hi-rustin) [reply .github/ISSUE_TEMPLATE/bug-report.yml]: Addressed.

**Review by amyangfei — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/question.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    attributes:
+      label: Before asking a question, make sure you have
+      options:
+        - label: Searched existing Stack Overflow questions
+          required: true
+        - label: Googled your question
+          required: true
+        - label: Searched open and closed [GitHub issues](https://github.com/pingcap/ticdc/issues?q=is%3Aissue)
+          required: true
+        - label:  Read the [documentation]((https://pingcap.com/docs/dev/reference/tools/ticdc/ticdc/))
```
Comment (amyangfei) [inline .github/ISSUE_TEMPLATE/question.yml:16]: syntax error, and seems the link is deprecated

**Comment by hi-rustin** [comment]
/cc @overvenus  @asddongmen 

Could you please take a look? Thanks!

File: .github/ISSUE_TEMPLATE/question.yml (hunk 3 above)
Comment (hi-rustin) [reply .github/ISSUE_TEMPLATE/question.yml:16]: Fixed! Good catch!

**Review by asddongmen — APPROVED** [review]

**Comment by hi-rustin** [comment]
/assign @amyangfei 

Could you please help me trigger the merge process? Thanks!

**Pull request merged**